ignition.spec

```diff
@@ -16,10 +16,9 @@ Patch3: 0002-build_releases-Allow-setting-VERSION-and-fallback-to.patch
 
 %if 0%{?fedora}
 BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang >= 1.6.2}
```

Comment:
Hum...doesn't this conditional already work on its own now?  I bet it can now just be:

`BuildRequires: %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang >= 1.6.2}`

Reply:
Ahh yeah, let me try that out.

Reply:
Yes, that works. Updated below. :arrow_down: